File: src/main/java/busanVibe/busan/global/apiPayload/exception/handler/ExceptionHandler.java
package busanVibe.busan.global.apiPayload.exception.handler;

import busanVibe.busan.global.apiPayload.code.BaseErrorCode;
import busanVibe.busan.global.apiPayload.exception.GeneralException;

public class ExceptionHandler extends GeneralException {
    public ExceptionHandler(BaseErrorCode code) {
        super(code);
    }
}

File: src/main/kotlin/busanVibe/busan/global/apiPayload/exception/handler/ExceptionHandler.kt
package busanVibe.busan.global.apiPayload.exception.handler

import busanVibe.busan.global.apiPayload.code.BaseErrorCode
import busanVibe.busan.global.apiPayload.exception.GeneralException

class ExceptionHandler(code: BaseErrorCode): GeneralException(code) {



}